docs/source/api/tutorials/In Depth 3 - Drawing DiHypergraphs.ipynb

```diff
@@ -13,7 +13,7 @@
    "id": "ea8aad6a",
```

Comment:
<p>The default cmap for hyperedges is now "Blues", which is different from all other functions where it is "crest". Any reason for that? I would maybe keep "crest" for consistency.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgi-org/xgi/pull/492/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>